instrumentation/spring/spring-webflux/spring-webflux-5.3/library/src/main/java/io/opentelemetry/instrumentation/spring/webflux/v5_3/server/SpringWebfluxNetAttributesGetter.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.spring.webflux.v5_3.server;
+
+import io.opentelemetry.instrumentation.api.instrumenter.net.NetServerAttributesGetter;
+import io.opentelemetry.semconv.trace.attributes.SemanticAttributes;
+import javax.annotation.Nullable;
+import org.springframework.web.server.ServerWebExchange;
+
+enum SpringWebfluxNetAttributesGetter implements NetServerAttributesGetter<ServerWebExchange> {
+  INSTANCE;
+
+  @Override
+  public String getTransport(ServerWebExchange request) {
+    return SemanticAttributes.NetTransportValues.IP_TCP;
+  }
+
+  @Nullable
+  @Override
+  public String getHostName(ServerWebExchange request) {
+    String host = request.getRequest().getURI().getHost();
+    if (host.equals("127.0.0.1")) {
```

Comment:
This is a bit of a hack. It seems that later versions of spring return the `127.0.0.1` whereas earlier version return `localhost`. There was also no option to set the expected `net.host.name` for the test.

Happy to leave it like this, or happy to add an option on the test (which would have to depend on system property `testLatestDeps`)